### DISCUSSION TITLE
Unable to locate config.custom.php in the data folder

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5056] [Q&A] [answered]
The comments in the config.default.php file suggest overriding the values in ./data/config.custom.php rather than the default ones. But as far as I can see, there is no such file in the data folder.

Has the file been moved elsewhere, removed (or) should I try modifying the default values?

The comment in the config.default.php file says this:
# Do not modify this file, which defines default values,
# but instead edit ./data/config.php after the install process is completed,
# or edit ./data/config.custom.php before the install process.

I have observed a similar issue with constants.php and config-user.default.php.

**Comment by Alkarex** [comment, marked as answer]
Hello,
You are supposed to create those local files yourself, using the same model. A blank file looks like that:

```php
<?php

return array(

);

```

**Reply by Frenzie** [reply]
@Alkarex If the code never creates such a file then it should say "create" instead of "edit" though. ;-)

**Reply by Frenzie** [reply]
Or maybe something like "copy and edit".
